AI/BattleAI/BattleAI.cpp

```diff
@@ -280,7 +325,47 @@ BattleAction CBattleAI::goTowards(const CStack * stack, const battle::Unit * ene
 
 BattleAction CBattleAI::useCatapult(const CStack * stack)

```

Comment:
Is this function still unused?

Reply:
Confirmed. Before when AI with balistics the balista was just doing nothing